quickstarts/bagisto/config.yml

```
@@ -0,0 +1,49 @@
+id: 9d758ec4-a261-06ef-be55-2e1f9cc55f2e
+slug: bagisto
+title: Bagisto The Laravel eCommerce
+
+description: |
+  ## Bagisto 
+  Bagisto is a E-Commerce framework built on top of the hottest opensource technologies such as Laravel, a PHP framework and Vue.js, a progressive Javascript framework.
+
+  Bagisto is a viable option to reduce your time, cost and workforce for building online stores or migrating from a physical store to an online platform.
+
+  It is suitable for all small or large E-Commerce business demands using a simple set-up procedure.
+  Built on top of Laravel and equipped with an easy product information management.
+
+  The framework is very flexible and easy to use even for non-tech users. Bagisto features an administration panel with a dashboard, sales data, a catalog for products and customer management.
+
+  To learn more about Bagisto's features and try a demo, check out our website https://bagisto.com.
+  Get quickly updated on the current version and recently released features, see Bagisto roadmap.
+  You can get started with the source code by checking out the repo on GitHub at bagisto/bagisto.
+
+summary: |
+  Bagisto is an open-source eCommerce framework built on Laravel that lets users manage multi-tenant businesses and POS processes.
+
+level: Community
+
+authors:
+  - Suraj Kashyap
+
+keywords:
+  - bagisto
+  - webkul
+  - laravel
+  - open-source
+  - vuejs
+  - e-commerce
+  - php
+  - bagisto
+  - language agent
+  - most popular
+
+documentation:
+  - name: Installation docs
+    url: https://devdocs.bagisto.com/1.x/introduction/
+    description: Bagisto installation docs
+
+icon: logo.png
+website: https://bagisto.com/en/
+
+dataSourceIds:
+  - bagisto
```

Hey @suraj-webkul! Since this seems to run on PHP under the hood, the PHP data source should be good enough to get us going. The data source represents the underlying technology that is sending the data to new relic.

```suggestion
  - php
```